.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
```

Comment:
this is very cool! i actually didn't know about the template config option!

Reply:
Yeah, I found it when scoping out how other repos set up their templates! The link seemed to behave exactly how the issue had suggested it to. Glad I was able to help with this 😄